mir_eval/util.py

```diff
@@ -621,9 +621,120 @@ def match_events(ref, est, window):
     # Construct the graph input
     G = {}
     for ref_i, est_i in zip(*hits):
-        if ref_i not in G:
-            G[ref_i] = []
-        G[ref_i].append(est_i)
+        if est_i not in G:
+            G[est_i] = []
+        G[est_i].append(ref_i)
+
+    # Compute the maximum matching
+    matching = sorted(_bipartite_match(G).items())
+
+    return matching
+
+
+def match_notes(ref_intervals, ref_pitches, est_intervals, est_pitches,
```

Comment:
So, I actually think this should probably just go in your new `precision_recall_f1` function.  The main idea behind having `match_events` is that it's called by a few different metric functions from different tasks.  I don't think the same would be true of `match_notes`.  Please correct me if I'm wrong.